### ISSUE TITLE
Get first and last measurement from defined period

### ISSUE THREAD

**Issue by mkasztelnik** [ismop/dap#15]
To display fiber view in GUI I need to get `last` sensor value from given time interval. What is more to distinguish when first value from this sensor was registered in DAP I need to get `first` measurement from given interval (when start time will be unix time 0).

It would be nice to have something like

```
https://dap/api/v1/measurements?limit=last
https://dap/api/v1/measurements?limit=first
```

All measurements should be returned in the measurement time order.

**Comment by mkasztelnik** [comment]
I was not precise enough in this issue description. I would like to have `last` or `first` value for **all** given timelines, so for example following query:

```
https://dap/api/v1/measurements?profile_id=1,2&limit=last
```

Should return 2 values (last value for each profile).

**Comment by mkasztelnik** [comment]
What is more current implementation returns corrupted elements, e.g. when in given time interval there is no measurements following response is returned:

```json
{ "measurements": [null] }
```

**Comment by bwilk** [comment]
implented as limit = each_timeline_last parameter

**Comment by nowakowski** [comment]
@bwilk I see what you did there but I was working on this in parallel and I think I have a better solution - right now both `limit=first` and `limit=last` return 1 measurement per timeline. I believe #33 is better than your commit. Please review and merge if you agree.

**Comment by bwilk** [comment]
Yours should be a bit more efficient. In fact this way you can easily extend this solution to return last "n" for each timelines - consider that. I' merging

**Comment by bwilk** [comment]
I'm not  :-). There are some conflicts but you have my thumb up.

**Comment by nowakowski** [comment]
@bwilk Thanks!

**Comment by nowakowski** [comment]
Replaced @bwilk 's changes with mine and merged with master. Invoke `limit=first` or `limit=last` to obtain 1 measurement per timeline (the first or last one respectively). An empty array is returned if no measurements match the search criteria. Deployed on production. Closing ticket.
